scripts/core/ui/components/UserPopup/UserPopup.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {showPopup} from 'core/ui/components/popupNew';
+import {IUser} from 'superdesk-api';
+import {UserAvatar} from 'apps/users/components/UserAvatar';
+
+interface IProps {
+  user: IUser;
+  mentionName: string;
+}
+
+const UserPopup = ({mentionName, user}: IProps) => {
+    const renderPopup = (referenceElement: HTMLElement) => {
+        showPopup(
+            referenceElement,
+            'left-end',
+            ({closePopup}) => (
+                <div className="user-popup" onMouseLeave={closePopup} style={{display: 'block'}}>
```

Comment:
`onMouseLeave` should also be put on the button that is triggering opening of the popup. Otherwise it doesn't close even if mouse leaves sidebar

![2022-03-10 16-59](https://user-images.githubusercontent.com/2076953/157702434-d64bd26d-6231-451d-8bc1-64e0f433bf75.gif)